src/ophyd_async/epics/adsimdetector/_sim.py

```diff
@@ -56,3 +56,52 @@ def __init__(
             name=name,
             config_sigs=config_sigs,
         )
+
+
+class ContAcqSimController(adcore.ADBaseContAcqController[SimDriverIO]):
+    """Controller for simulated Areadetector in continuous acquisition mode."""
+
+    def __init__(
+        self,
+        cb_plugin_prefix: str,
+        driver: SimDriverIO,
+    ):
+        super().__init__(cb_plugin_prefix, driver)
+
+    def get_deadtime(self, exposure: float | None) -> float:
+        return 0.001
+
+
+class ContAcqSimDetector(adcore.ContAcqAreaDetector[ContAcqSimController]):
```

Comment:
... couldn't we just derive from `adcore.AreaDetector` here?

Reply:
Originally I was going to follow the same model we did for the default classes, since different drivers can have slightly different behavior w.r.t. setting exposure while running for example, but I decided to now just make a class that covers 99% of cases that just checks to make sure we are already in the right mode. If for whatever reason people need to be able to change the modes they can make another class, or use `load_settings`.